charts/argo-rollouts/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-version: 0.0.17
-appVersion: 1.1.0
+version: 1.2.2-ak.0.0
```

Comment:
Is it intentional to have ak.0.0 instead of ak.0?

Reply:
Yes, it's because we need two parts of the versioning. Broken down it is:

* `1.2.2` - matches upstream release
* `-ak.0` - akuity specific version. The `.0` in the `-ak.0` indicates it is even with upstream. If `> 0` it indicates it is an akuity specific patch ontop of upstream `v1.2.2`
* `.0` - the last digit is if we need to make helm chart changes but are no changes to actual image (e.g. we introduce a new helm option or fix a bug in the helm chart)

Reply:
Sounds good!